receiver/apachereceiver/metadata.yaml

```diff
@@ -76,6 +76,129 @@ metrics:
       monotonic: true
       aggregation: cumulative
     attributes: [ server_name ]
+  apache.bytes_per_request:
+    enabled: true
+    description: The average number of bytes per request.
+    unit: By
+    sum:
+      value_type: double
+      input_type: string
+      monotonic: false
+      aggregation: delta
+    attributes: [ server_name ]
+  apache.bytes_per_second:
+    enabled: true
+    description: The average number of bytes per second.
+    unit: By
+    sum:
+      value_type: double
+      input_type: string
+      monotonic: false
+      aggregation: delta
+    attributes: [ server_name ]
+  apache.cpu_children_system:
+    enabled: true
+    description: Jiffies used in system mode by children processes.
+    unit: s
+    gauge:
+      value_type: double
+      input_type: string
+    attributes: [ server_name ]
+  apache.cpu_children_user:
+    enabled: true
+    description: Jiffies used in user mode by children processes.
+    unit: s
+    gauge:
+      value_type: double
+      input_type: string
+    attributes: [ server_name ]
+  apache.cpu_system:
+    enabled: true
+    description: Jiffies used in system mode.
+    unit: s
+    gauge:
+      value_type: double
+      input_type: string
+    attributes: [ server_name ]
+  apache.cpu_user:
+    enabled: true
+    description: Jiffies used in user mode.
+    unit: s
+    gauge:
+      value_type: double
+      input_type: string
+    attributes: [ server_name ]
+  apache.cpu_load:
+    enabled: true
+    description: Current load of the CPU.
+    unit: "{load}"
+    sum:
+      value_type: double
+      input_type: string
+      monotonic: false
+      aggregation: delta
+    attributes: [ server_name ]
+  apache.duration_per_request:
```

Comment:
I'm not sure what do you mean by "counter", so I'm assuming that you're referring to another metric that is an integer value (for example, `bytes_per_request` can be expressed as `apache.traffic / apache.requests`). 
bytes per request, bytes per second: yes, I referred to them in two other comments
`apache.request.average_time` (`DurationPerReq`) - yes, it's `apache.total_duration / apache.requests`
`apache.load.*` - I guess not, since they refer to the last X minutes and we don't have that info
`apache.request.rate` (`ReqPerSec`) - yes, it's `apache.requests / apache.uptime`

So, I guess that you suggest to remove the metrics than can be calculated basing on other metrics?

Reply:
I should have said "sum" rather than "counter", but you got the idea. (These are often interchangeable terms.)

Yes, typically we would keep only the sums. The rates, as calculated, are not terribly useful anyways since they are anchored at system startup. The longer the system is running, the less useful the average becomes, as it no longer necessarily reflects the current state of the system. For example, on a system with 0 traffic for several hours, these rate metrics will hardly register a brief burst of traffic.

On the other hand, it is possible to calculate rates that much more accurately reflect recent activity. A few caveats apply, but roughly:
```
recentRequests = (latest) apache.requests - (next latest) apache.requests
recent uptime = (latest) uptime - (next latest) uptime
recent request rate = recent requests / recent uptime
```

Typically, these scrapers are concerned with obtaining the counts, and backends are left to derive rates.